src/js/tracks/text-track-list.js

```diff
@@ -111,6 +111,7 @@ TextTrackList.prototype.removeTrack_ = function(rtrack) {
   for (let i = 0, l = this.length; i < l; i++) {
     if (this[i] === rtrack) {
       track = this[i];
+      track.off();
```

Comment:
It looks like one of the tests caught that scenario. I'm checking for track.off existence since it seems like the input is still a TextTrack even though it's an element.